src/Error/ExceptionTrap.php

```diff
@@ -93,6 +96,9 @@ public function renderer(Throwable $exception, $request = null)
 
         /** @var class-string|callable $class */
         $class = $this->_getConfig('exceptionRenderer');
+        if (!$class) {
+            $class = $this->chooseRenderer();
+        }
```

Comment:
When is this not set by chooseRenderer() in the constructor?

Reply:
There is the possibility that a user could use `setConfig()` to inject invalid data. I could remove the constructor logic as it is redundant.

Reply:
Yea, we might as well just chose it here and leave config as-is in the constructor.